wiki/Skinning/Interface/en.md

```diff
@@ -939,8 +938,8 @@ _`fail-background.png`_ (transparent; not included)
 
 **Notes:**
 
-- If one is not skinned, the default is to dim the background.
-- This element will strech to fit height.
+- If one is not skinned, the background of the map will be dimmed by default.
```

Comment:
ditto; rephrase to `If not skinned, the background of the map will be dimmed by default.` or similar